<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SeriesWidget</class>
 <widget class="QWidget" name="SeriesWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>491</width>
    <height>440</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <property name="documentMode">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>General</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QFormLayout" name="formLayout">
         <property name="fieldGrowthPolicy">
          <enum>QFormLayout::ExpandingFieldsGrow</enum>
         </property>
         <item row="0" column="0">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Series Type:</string>
           </property>
           <property name="buddy">
            <cstring>editType</cstring>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="KComboBox" name="editType"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Title:</string>
           </property>
           <property name="buddy">
            <cstring>editTitle</cstring>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="StringEdit" name="editTitle"/>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Issn:</string>
           </property>
           <property name="buddy">
            <cstring>editIssn</cstring>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="StringEdit" name="editIssn"/>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Rating:</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="KRatingWidget" name="editRating"/>
         </item>
        </layout>
       </item>
       <item>
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="ListPartsWidget" name="listPartWidget" native="true"/>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Note</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="ListAnnotationsWidget" name="editAnnot" native="true"/>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>380</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>KRatingWidget</class>
   <extends>QFrame</extends>
   <header>kratingwidget.h</header>
  </customwidget>
  <customwidget>
   <class>StringEdit</class>
   <extends>QLineEdit</extends>
   <header>propertywidgets/stringedit.h</header>
   <slots>
    <signal>widgetEnabled(bool)</signal>
    <signal>widgetShown(bool)</signal>
   </slots>
  </customwidget>
  <customwidget>
   <class>ListPartsWidget</class>
   <extends>QWidget</extends>
   <header>sidebar/listpartswidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ListAnnotationsWidget</class>
   <extends>QWidget</extends>
   <header>sidebar/listannotationswidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>editType</tabstop>
  <tabstop>editTitle</tabstop>
  <tabstop>editIssn</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>editTitle</sender>
   <signal>widgetEnabled(bool)</signal>
   <receiver>label</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>101</x>
     <y>66</y>
    </hint>
    <hint type="destinationlabel">
     <x>70</x>
     <y>63</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>editIssn</sender>
   <signal>widgetEnabled(bool)</signal>
   <receiver>label_2</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>143</x>
     <y>97</y>
    </hint>
    <hint type="destinationlabel">
     <x>80</x>
     <y>90</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>editIssue()</slot>
  <slot>addIssue()</slot>
  <slot>removeIssue()</slot>
 </slots>
</ui>
